cluster_management/src/main/java/com/pinterest/rocksplicator/Utils.java

```diff
@@ -146,6 +149,18 @@ public static Admin.Client getAdminClient(String host, int adminPort) throws TTr
     return new Admin.Client(new TBinaryProtocol(sock));
```

Comment:
So, any call to this api will have socket leak then? do we need to fix them?

Reply:
yeah, we need to and I plan to do a follow up PR. Note that other places are not causing problems currently because they are using the admin client in StateTransition, so it's usually one off, unlike `PartitionStateUpdater` which happens continuously.